cordova-lib/src/cordova/plugin.js

```diff
@@ -308,8 +301,8 @@ module.exports = function plugin(command, targets, opts) {
 };
 
 function getVersionFromConfigFile(plugin, cfg){
-    var feature = cfg.getFeature(plugin); 
-    return feature && (feature.params.url || feature.params.installPath || feature.params.version);
+    var pluginEntry = cfg.getPlugin(plugin);
+    return pluginEntry && pluginEntry.version; 
```

Comment:
This wouldn't work if config.xml contains a url or a folder.
This line should be changed to : return pluginEntry && pluginEntry.version || pluginEntry.src;